charts/incubator/mineos/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+kubeVersion: ">=1.16.0-0"
+name: mineos
+version: "0.0.1"
+appVersion: "latest"
```

Comment:
```suggestion
version: 0.0.1
appVersion: "latest"
```

Reply:
@stavros-k  and I tried so many iterations, it always fails near the same part, I didnt mean to leave version as a string, but app version needed to updated since "1.35.1" was not a value found in docker for mineos. the format of the the chart yaml changed and was all over the place, and I didnt do that, I was putting it back to the same standard as the other chart files.